doc/guides/building-node-with-ninja.md

```diff
@@ -22,27 +25,21 @@ The bottom line will change while building, showing the progress as
 `[finished/total]` build steps. This is useful output that `make` does not
 produce and is one of the benefits of using Ninja. Also, Ninja will likely
 compile much faster than even `make -j4` (or
-`-j<number of processor threads on your machine>`).
+`-j<number of processor threads on your machine>`), you can still pass the
+number of processes to run for [Ninja][] using the environment variable `JOBS`,
+and this will be the equivalent to the `-j` parameter in the regular `make`:
```

Comment:
```suggestion
`-j<number of processor threads on your machine>`). You can still pass the
number of processes to run for Ninja using the environment variable `JOBS`.
This will be the equivalent to the `-j` parameter in the regular `make`:
```